tests/e2e_commands.txt

```diff
@@ -37,6 +38,7 @@ netexec smb TARGET_HOST -u LOGIN_USERNAME -p LOGIN_PASSWORD KERBEROS --get-file
 ##### SMB PowerShell
 netexec smb TARGET_HOST -u LOGIN_USERNAME -p LOGIN_PASSWORD KERBEROS -X ipconfig
 netexec smb TARGET_HOST -u LOGIN_USERNAME -p LOGIN_PASSWORD KERBEROS -X ipconfig --exec-method atexec
+netexec smb TARGET_HOST -u LOGIN_USERNAME -p LOGIN_PASSWORD KERBEROS -X ipconfig --exec-method atexec --run-task-as Admin
```

Comment:
Admin ? 🤔

Reply:
You can specify whoemever you want "WHITEFLAG\Administrateur" "WHITEFLAG\Admin" "Administrateur" (<-- local admin)

Reply:
better put a default one (most common one) `administrator` otherwise test will always fail with admin

Reply:
Ah got it